### PR TITLE
Rya-446 RPM install

### PR DESCRIPTION
**Pull request by kchilton2**
## Description
Implemented and tested the Rya Streams Query Manager RPM installer on CentOS 7

### Links
https://issues.apache.org/jira/browse/RYA-466

### CODE REVIEW

File: extras/rya.streams/query-manager/src/main/java/org/apache/rya/streams/querymanager/QueryManager.java [hunk 1]
```diff
@@ -51,205 +61,790 @@
...
+    private final ExecutorService executor = Executors.newFixedThreadPool(2);
 
     /**
      * Creates a new {@link QueryManager}.
      *
      * @param queryExecutor - Runs the active {@link StreamsQuery}s. (not null)
      * @param source - The {@link QueryChangeLogSource} of QueryChangeLogs. (not null)
-     * @param scheduler - The {@link Scheduler} used to discover query changes
-     *      within the {@link QueryChangeLog}s (not null)
+     * @param blockingValue - How long blocking operations will try before looping. (not null)
```
Comment (isper3at) [inline extras/rya.streams/query-manager/src/main/java/org/apache/rya/streams/querymanager/QueryManager.java]: longs can't be null.  non-negative check?

File: extras/rya.streams/query-manager/src/main/scripts/systemd/rya-streams-query-manager.service [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# specific language governing permissions and limitations
+# under the License.
+#
+[Unit]
+Description=Rya Streams Query Manager Service
+After=network.target
+
+[Service]
+Type=forking
+PIDFile=/var/run/rya-streams-query-manager.pid
```
Comment (isper3at) [inline extras/rya.streams/query-manager/src/main/scripts/systemd/rya-streams-query-manager.service:25]: if we're hardcoding the PID here, we might as well in the service scripts

Comment (kchilton2) [reply extras/rya.streams/query-manager/src/main/scripts/systemd/rya-streams-query-manager.service:25]: It already is hard coded in the service scripts.

Comment (isper3at) [reply extras/rya.streams/query-manager/src/main/scripts/systemd/rya-streams-query-manager.service:25]: saw the $NAME and was curious

File: extras/rya.streams/query-manager/src/main/java/org/apache/rya/streams/querymanager/QueryManager.java (hunk 1 above)
Comment (kchilton2) [reply extras/rya.streams/query-manager/src/main/java/org/apache/rya/streams/querymanager/QueryManager.java]: done

**Pull request merged**